### PR TITLE
Removing padding-bottom overriding paper-input styles

### LINKED ISSUE #157: Vertical alignment together with paper-input in IE11
Using `display: inline-block;` for both vaadin-combo-box and paper-input side-by-side causes them to be vertically misaligned in Windows 8.1, IE11:

screen shot 2016-02-19 at 11 15 13

Caused by overriding the padding-bottom of paper-input-container to zero in our styles.

### PR DESCRIPTION
**Pull request by manolo**
Fixes #157

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box/216)

### CODE REVIEW

**Comment by manolo** [comment]
This is how it looks now:

screen shot 2016-04-12 at 14 02 07

**Comment by platosha** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-:-KF9MWHrvzO1yrdFKJr7:1417088721)*

**Comment by jounik** [comment]
Notice that the dropdown position is affected by this. It should still be connected wih the input underline.

**Comment by platosha** [comment]
Oh, right. :lgtm_cancel: 

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-:-KF9PdZ65rdWsHpgcdVA:2142675885)*

**Comment by manolo** [comment]
Fixed position

screen shot 2016-04-12 at 14 02 07

File: vaadin-combo-box.html [hunk 1]
```diff
@@ -63,7 +64,7 @@
 
     paper-input-container {
       position: relative;
-      padding-bottom: 0;
+      padding-bottom: 0px;
```
Comment (jounik) [inline vaadin-combo-box.html]: Unnecessary change

**Comment by platosha** [comment]
I feel like we should get rid of padding in the input container completely, but set `padding: 8px 0;` for the combo-box instead. To be discussed more tomorrow.

Not a direct blocker, but would be nice to change it in this PR as well, if we decide to.

**Comment by platosha** [comment]
:lgtm: Verified in IE11


 

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-:-KFDseJ7cDQpD5Vd1Ml0:-1518826644)*

**Comment by manolo** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[vaadin-combo-box.html, line 67 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-KF9Lcn9pL_r6xgPLHxn-r2-67:-KFDt-k-FSV4Si_9MwHA:-718130138) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/fecb7cd29df92eab7ea562bd0c7773d512373364/vaadin-combo-box.html#L67)):*
@platosha  suggested it. Anton could you answer?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216)*

**Comment by platosha** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[vaadin-combo-box.html, line 67 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-KF9Lcn9pL_r6xgPLHxn-r2-67:-KFDtIugwSlUVp0sIAkP:1862491173) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/fecb7cd29df92eab7ea562bd0c7773d512373364/vaadin-combo-box.html#L67)):*
This has changed into a necessary change already.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216)*

**Comment by tomivirkki** [comment]
*[vaadin-combo-box.html, line 58 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-KFDvLB6tWigxYid9Mno:-KFDvLB6tWigxYid9Mnp:-1633920812) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/129fed9083287b7a4ae25154da02a8df3478020c/vaadin-combo-box.html#L58)):*
Feels unorthodox that a web-component affects it's host component's box model. Things like this should be kept inside the local dom. Why is this better than not having the padding declared for paper-input-container?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216)*

**Comment by platosha** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[vaadin-combo-box.html, line 58 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-KFDvLB6tWigxYid9Mno:-KFDwOXPIcbeT4FwSABg:363595640) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/129fed9083287b7a4ae25154da02a8df3478020c/vaadin-combo-box.html#L58)):*
2 things.

Having no padding in paper-input-container makes the dropdown positioning code more straightforward. Our aim is to glue the dropdown to the edge of the input container's content box. If there's no padding, we can just use getBoundingClientRect() data. When there's a padding, we need to get the padding values and bother with subtracting them. While doable, this feels too much for a problem that could be resolved with a simple css line.

From application developer's point of view, it's easier to customize the padding on the hosting element with styles, then if it's hidden inside. So I see this padding being exposed on the hosting element as an improvement rather then a drawback.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216)*

**Comment by jounik** [comment]
I can approve the change visually (moving the dropdown closer to the input when positioned on the top), but I’m not requiring it. I’ll let you settle the implementation details.

**Comment by tomivirkki** [comment]
*[vaadin-combo-box.html, line 58 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216#-KFDvLB6tWigxYid9Mno:-KFE2D0Ue6gHBCzg9N-P:-1826068044) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/129fed9083287b7a4ae25154da02a8df3478020c/vaadin-combo-box.html#L58)):*
Ok I kind of expected vaadin-combo-box's default box model to be in line with that of paper-input.
For example if you place these input elements side-by-side in a container and have a rule like

container * {
 padding: 10px;
}

...the elements will be out of sync? You'll need to provide a special tule for vaadin-combo-box that has 8px more padding-top/bottom than other paper input components.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/216)*
